File: Bubble_Sort/Bubble_Sort.py
# function for bubble sort
def Bubble_Sort(list):
    for i in range(0, len(list) - 1):
        for j in range(0, len(list) - i - 1):
            # do swapping
            if list[j] > list[j + 1]:
                list[j], list[j + 1] = list[j + 1], list[j]

# function to print list
def Print_list(list):
    for i in range(0, len(list)):
        print(list[i], end = " ")

    print()

list = [2, 4, 3, 1, 6, 8, 4]

Bubble_Sort(list)

Print_list(list)

# Output
# 1 2 3 4 4 6 8

File: Bubble_Sort/Bubble_Sort.cs
using System;

namespace Bubble_Sort
{
    class Program
    {
        // Function for bubble sort
        public static void BubbleSort(int[] array, int size)
        {
            int temp;

            for(int i = 0; i < size - 1; i++)
            {
                for(int j = 0; j < size - i - 1; j++)
                {
                    if(array[j] > array[j + 1])
                    {
                        temp = array[j];
                        array[j] = array[j + 1];
                        array[j + 1] = temp;
                    }
                }
            }

        }

        // Function to print array
        public static void Print_Array(int[] array, int size)
        {
            for(int i = 0; i < size; i++)
                Console.Write(Convert.ToString(array[i]) + " ");

            Console.WriteLine();
        }

        static void Main(String[] args)
        {
            int[] array = { 2, 4, 3, 1, 6, 8, 4 };
            BubbleSort(array, 7);
            Print_Array(array, 7);
            Console.WriteLine();
            Console.ReadLine();
        }
    }
}

// Output
// 1 2 3 4 4 6 8
